README.md

```diff
@@ -1 +1,26 @@
+# Burger Menu for Recipients
+
+## Description  
+In the landing page recipients have a burger menu in the top right corner. Once a user clicks on it a menu appears from right to left. In this menu users can find following links:
+
+*  Perfil
+*  Notificaciones
+*  Historial de recogidas
+*  Punto de recogida
+*  Hoja de derivación
+*  Ayuda
+*  Eliminar cuenta
+*  Log out
+
+## Definition of Done 
+By clicking on the "X" icon in the top right corner the menu disappear from left to right showing the landing page again.
```

Comment:
```suggestion
# Fesbal Frontend

This is a React application for the FESBAL NGO to manage their donations.

## Components

In this section you can find descriptions on the components that have been developed, including some details on their business logic

### Burger Menu

In the landing page recipients have a burger menu in the top right corner. Once a user clicks on it a menu appears from right to left. In this menu users can find following links:

*  Perfil
*  Notificaciones
*  Historial de recogidas
*  Punto de recogida
*  Hoja de derivación
*  Ayuda
*  Eliminar cuenta
*  Log out

By clicking on the "X" icon in the top right corner the menu disappear from left to right showing the landing page again.
```

Thanks for updating the readme! I would recommend making it more generic. In my opinion, the first section should be talking about the project in general terms, then you can add a description of how your specific container works